content/blog/blog-post-08.md

```diff
@@ -0,0 +1,52 @@
+---
+authors: ["kokkos-team"]
+title: "Kokkos Joins High Performance Software Foundation"
+date: 2024-05-20
+tags: ["blog"]
+thumbnail: img/blog/2024/kokkos-hpsf.png
+---
+
+Kokkos is among the six initial technical projets of the newly created High Performance Software Foundation ([HPSF](https://hpsf.org)).
+HPSF is set up as an umbrella organization within the Linux Foundation to foster collaboration around the development and support of its associated technical projects.
+Conceived through an initial collaboration between the Spack and the Kokkos team, this effort has brought together institutions that want to foster a more collaborative effort of developing and maintaining key open-source projects in the space of high performance computing and related compute intensive areas.
+
+Historically the HPC community has been characterized by a large amount of fragmentation, with many similar or even identical capabilities developed at different institutions.
+The lack of reliable software support funding mechanism in the scientific community led to a reluctance of relying on third party libraries, and the lack of community governance models hindered the establishment of joint multi-institutional development efforts.
```

Comment:
This line is pretty passive. Maybe we could be more active and say something like:

> Many existing projects were reluctant to rely on third party software due to uncertain funding. Without community governance models, institutions found it difficult to collaborate effectively in join development projects.

Or something like that.